tests/client/amo/components/TestLink.js

```diff
@@ -90,4 +90,23 @@ describe('<Link />', () => {
       { pathname: 'test' }
     );
   });
+
+  it('ignores `href` if not a string type', () => {
+    const root = render({ base: null, href: null });
+
+    assert.ok(findRenderedComponentWithType(root, Link));
+  });
+
+  it('passes `href` without leading slash without base', () => {
+    const root = render({ base: '/base/', href: 'test' });
+
+    assert.notInclude(findDOMNode(root, 'a').href, '/base/');
+    assert.include(findDOMNode(root, 'a').href, 'test');
+  });
+
+  it('normalizes the href path with a string', () => {
+    const root = render({ base: '/foo/', href: '/test' });
+
+    assert.include(findDOMNode(root, 'a').href, '/foo/test');
+  });
```

Comment:
It would be nice to add a test that ensures the children of `<a ...>{children}</a>` are rendered.